tests/models/modules/test_masked_autoencoder_timm.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+
+import torch
+
+from lightly.models import utils
+from lightly.utils import dependency
+
+if dependency.timm_vit_available():
+    from timm.models.vision_transformer import VisionTransformer, vit_base_patch32_224
+
+    from lightly.models.modules import MAEDecoderTIMM, MaskedVisionTransformerTIMM
+
+    class TestMAEBackboneTIMM(unittest.TestCase):
```

Comment:
Let's fix this in the next PR: This should be `TestMaskedVisionTransformerTIMM` and in `test_masked_vision_transformer_timm.py`

Reply:
hm but we have also the tests for the decoder with TIMM. I can rename the class to `TestMaskedVisionTransformerTIMM `, but I would not change the name of the file TBH

Reply:
Please move it to a new file. If you write tests for functions/classes from `dir/something.py` then the corresponding tests should go to `tests/dir/test_something.py`

Reply:
That is what I have. The test for `lightly/models/modules/masked_autoencoder_timm.py` is in `tests/models/modules/test_masked_autoencoder_timm.py`